plugins/modules/proxmox_backup.py

```diff
@@ -0,0 +1,751 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright John Berninger (@jberning) <john.berninger at gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_backup
+short_description: Create, delete, or update Proxmox VE backup jobs
+version_added: 9.1.0
+description:
+  - Allows you to perform some supported operations on a backup job in a Proxmox VE cluster.
+author: Dylan Leverrier (@zerchevack)
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: full
+options:
+  state:
+    description:
+      - Set to V(present) to create or update job.
+      - Set to V(absent) to delete job.
+    choices: ['present', 'absent']
+    type: str
+    required: true
+  all:
+    description:
+      - Backup all known guest systems on this host.
+      - Can not be use with O(vmid) and O(pool) in same job
+    type: bool
+  bwlimit:
+    description:
+      - Limit I/O bandwidth (in KiB/s).
+    type: int
+  comment:
+    description:
+      - Description for the Job.
+    type: str
+  compress:
+    description:
+      - >
+        If you choice a renote storage (like Proxmox Backup Server storage) the V(zstd) will be set automatically and this the only available value.
+        If you choice a local storage you can choice between V(gzip), V(lzo) and V(zstd).
+    choices: ['gzip', 'lzo', 'zstd']
+    type: str
+  dow:
+    description:
+      - Day of week selection.
+    type: str
+  dumpdir:
+    description:
+      - Store resulting files to specified directory.
+    type: str
+  enabled:
+    description:
+      - Enable or disable the job.
+    type: bool
+  exclude:
+    description:
+      - Exclude specified guest systems (assumes --all)
+    type: str
+  exclude_path:
+    description:
+      - >
+        Exclude certain files/directories (shell globs).
+        Paths starting with '/' are anchored to the container's root,
+        other paths match relative to each subdirectory.
+    choices: ['ignore', 'on']
+    type: str
+  fleecing:
+    description:
+      - Options for backup fleecing (VM only).
+    type: str
+  id:
+    description:
+      - Required if O(state=absent).
+      - If O(state=present), it allow you to set a pattern of id (Example 0(backup-12345678-9123)) if it not set an ID will be generate automaticly.
+      - Rerquired if O(state=present) and you want to update a existing job.
+    type: str
+  ionice:
+    description:
+      - Set IO priority when using the BFQ scheduler.
+      - For snapshot and suspend mode backups of VMs, this only affects the compressor.
+      - A value of 8 means the idle priority is used,
+      - otherwise the best-effort priority is used with the specified value.
```

Comment:
```suggestion
      - A value of 8 means the idle priority is used,
        otherwise the best-effort priority is used with the specified value.
```
Please don't spread sentences over two paragraphs.

Reply:
fix in 900e035f651b9a87963eadaf56385fa61fde9433